Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Borgs/borgs_tools.yml

```diff
@@ -82,6 +85,37 @@
     tags:
     - DiscreteHealthAnalyzer
 
+- type: entity
+  parent: BaseItem
+  id: HandheldCrewMonitorBorg
+  name: handheld crew monitor
+  suffix: Robot, No Map
+  description: A batteryless hand-held crew monitor designed for robots which displays the status of the crew suit sensors.
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/handheldcrewmonitor.rsi
+    state: scanner
+  - type: ActivatableUI
+    key: enum.CrewMonitoringUIKey.Key
+    closeOnHandDeselect: false
+  - type: UserInterface
+    interfaces:
+      - key: enum.CrewMonitoringUIKey.Key
+        type: CrewMonitoringBoundUserInterface
+  - type: CrewMonitoringConsole
+  - type: DeviceNetwork
+    deviceNetId: Wireless
+    receiveFrequencyId: CrewMonitor
+  - type: WirelessNetworkConnection
+    range: 500
+  - type: DynamicPrice
+    price: 500
+  - type: StationLimitedNetwork
+  - type: ReverseEngineering
+    difficulty: 2
+    recipes:
+      - HandheldCrewMonitor
```

Comment:
Was this copied from the original crew monitor? This is also unnecessary for the same underlying reason that it doesn't need a Price: it won't be an item that's dropped.